nuclei/client/main.py

```diff
@@ -14,8 +14,10 @@
 
     from nuclei.client import utils
 except ImportError:
-    utils = None  # type: ignore
-    Image = None  # type: ignore
+    raise ImportError(
+        "Could not import one of dependencies [geopandas, numpy, pandas, polars, Image].  "
```

Comment:
Do we want to raise a custom `ImportError` for the client.utils?

Reply:
We could raise a custom error in the client.utils and catch that here. I'll make it a separate ticket